### PR TITLE
Upgrade/add multi module support

### PR DESCRIPTION
**Pull request by peter-madigan**
Adds multi-module support for light simulation

### CODE REVIEW

**Comment by peter-madigan** [comment]
Includes #77

File: larndsim/detector_properties/module0.yaml [hunk 1]
```diff
@@ -26,35 +27,38 @@ module_to_io_groups:
   1: [1,2,3,4]
```
Comment (soleti) [inline larndsim/detector_properties/module0.yaml:27]: I guess here it should be `1: [1,2]` right?

Comment (peter-madigan) [reply larndsim/detector_properties/module0.yaml:27]: Oh yeah, I guess I missed that

Comment (soleti) [reply larndsim/detector_properties/module0.yaml:27]: Can you change it in this PR?

Comment (peter-madigan) [reply larndsim/detector_properties/module0.yaml]: Done

**Pull request merged**